satellite/console/users.go

```diff
@@ -64,6 +64,18 @@ func (user *CreateUser) IsValid() error {
 	return errs.Combine()
 }
 
+// UserStatus - is used to indicate status of the users account
+type UserStatus int
+
+const (
+	// Inactive is a user status that he receives after registration
+	Inactive UserStatus = iota
```

Comment:
Specify the numbers explicitly, otherwise people may accidentally rename/move these things and break things.